tests/test_qaoa_qiskit.py

```diff
@@ -73,7 +73,8 @@ def test_parameterized_circuit():
     backend = AerSimulator(method="statevector")
 
     qc_param = get_parameterized_qaoa_circuit(N, terms, p)
-    qc1 = qc_param.bind_parameters(np.hstack([ramp["beta"], ramp["gamma"]]))
+    qc1 = qc_param.assign_parameters(np.hstack([ramp["beta"], ramp["gamma"]]))
+    # qc1 = qc_param.bind_parameters(np.hstack([ramp["beta"], ramp["gamma"]]))
```

Comment:
Can you please remove commented out code?

Reply:
Done!